src/utils/swapQuotes.ts

```diff
@@ -1,10 +1,27 @@
+import { asNumber, asString } from 'cleaners'
 import {
   EdgeAccount,
   EdgeCurrencyWallet,
   EdgePluginMap,
-  EdgeSwapQuote
+  EdgeSwapQuote,
+  EdgeSwapRequest
 } from 'edge-core-js'
 
+export const createFromSwapRequest = (
```

Comment:
Instead of "regular" function this should be a cleaner.
I'll show you tomorrow what I mean.